README.md

```diff
@@ -1 +1,45 @@
-# spot-electron-sdk
\ No newline at end of file
+# Spot Beacon Desktop SDK
+This SDK provides spot beacon functionality for desktop (electron) apps.
+
+# Usage
+1. Instantiate the SDK with (optional) configuration object.
+
+```
+const { BeaconEmitter } =  require('spot-electron-sdk');
+const emitter = new BeaconEmitter({
+    beaconUUID: string,
+    beaconDismissTimeoutSeconds: number,
+    reportIntervalMillisecs: number
+});
+```
+2. Subscribe to events
+
+```
+emitter.on('scanStart', () => {
+    console.log('Beacon scanning started.');
+});
+emitter.on('scanStop', () => {
+    console.log('Beacon scanning stopped.');
+});
+emitter.on('beacons', beacons  => {
+    console.log('Beacons', beacons);
+});
+emitter.on('bestBeacon', beacon  => {
+    console.log('Best beacon', beacon);
+});
+```
+# Reference
+## Config object
+`beaconUUID`: (Optional) The Beacon UUID to look for. Omit this to detect Jitsi beacons, but custom deployments should have their own ID.
+`beaconDismissTimeoutSeconds`: (Optional) The timeout in seconds that the SDK waits before dismissing (not reporting anymore) a beacon that it stopped detecting. Min: 5s.
+`reportIntervalMillisecs`: (Optional) The timeout in milliseconds the SDK reports newly detected devices (if any). Min: 2000ms.
```

Comment:
reportIntervalMs ?

Reply:
I like to write out these words, like seconds above. Maybe it's because of my Java background? Dunno :)